ios/FluentUI/Card/CardView.swift

```diff
@@ -399,51 +406,69 @@ open class CardView: UIView {
         delegate?.didTapCard?(self)
     }
 
-    private func labelHeight(for sizeCategory: UIContentSizeCategory) -> (CGFloat, CGFloat) {
-        var singleLineHeight: CGFloat = 0
-        var twoLineHeight: CGFloat = 0
+    private func labelHeight(for sizeCategory: UIContentSizeCategory, size: CardSize) -> CGFloat {
+        var height: CGFloat = 0
```

Comment:
This func can also be moved to the CardSize enum (passing only the UIContentSizeCategory as a param).